demo-app/src/main/java/com/gravatar/demoapp/ui/DemoGravatarApp.kt

```diff
@@ -186,7 +186,7 @@ private fun ProfileComponents(profileState: ComponentState<Profile>?, theme: The
         val defaultModifier = Modifier
             .background(gravatarTheme.colorScheme.surfaceContainer)
             .fillMaxWidth()
-            .padding(24.dp)
+            .padding(start = 24.dp, bottom = 10.dp, end = 24.dp, top = 21.dp)
```

Comment:
This adjusts the paddings to match the Figma. I think these should be included in our components - WDYT?

Reply:
> I think these should be included in our components

I'm not completely sure about adding paddings to our components. We've tried to make them as customizable as possible, so we didn't add them internally to allow SDK users to set whatever they want and keep the component as simple as possible. 🤔

Reply:
I think these could be the defaults unless others were specified. This way by default we end up with what our design shows and not no padding at all.

Reply:
I guess you are talking about initializing the modifier with the default padding. However, that's not a recommended practice:

<img width="600" alt="image" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/4f5be433-5d68-4068-8c4d-f53f9f888f3b">

[Here](https://stackoverflow.com/questions/73415055/why-was-i-warned-optional-modifier-parameter-should-have-a-default-value-of-mod) they give a good explanation about why it is not recommended.

Reply:
I see! Let's leave it for now as it's not part of this PR anyway but I think this might be worth discussing with the design.